tests/stress_tests/memcheck_tests/tests.cpp

```diff
@@ -106,3 +106,53 @@ INSTANTIATE_TEST_CASE_P(MemCheckTests, MemCheckTestSuite,
                         ::testing::ValuesIn(
                                 generateTestsParams({"devices", "models"})),
                         getTestCaseName);
+
+TEST_P(MemCheckTestSuite, inference_with_streams) {
+    const auto nstreams = 2;
```

Comment:
Can we parameterize it?

Reply:
Do we have a use cases where different number of streams lead to different behaviour? 

I'm not sure it's really required because in case if memory consumption is affected it should be presented in each case, so we may choose some default value